modules/migration-using-must-gather.adoc

```diff
@@ -5,13 +5,21 @@
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
 
 [id="migration-using-must-gather_{context}"]
-= Using must-gather to collect data
 
-You must run the `must-gather` tool if you open a customer support case on the link:https://access.redhat.com[Red Hat Customer Portal] for the {mtc-full} ({mtc-short}).
 
-The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migration-specific logs and data that are not collected by the default `must-gather` image.
 
-.Procedure
+= Collecting logs and metrics with the must-gather tool
+
+You can collect logs, metrics, and information about {project-short} custom resources by using the `must-gather` tool.
```

Comment:
This MTV attribute won't work for MTC. You have to use the correct doc attributes.

Reply:
Click "Resolve conversation" for outdated comments.